### PR TITLE
User/jeremy kelley/scan guidance

### PR DESCRIPTION
**Pull request by JeremyKelley**
Adding the first round of updates covering:
1) Permission scope guidance
2) Download guidance
3) Permission hierarchies

### CODE REVIEW

**Comment by daspek** [comment]
OPS Build status updates of commit _[7289559](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/72895595e0df8efcccbe7862bf9ec269542aa1c7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[7289559](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/72895595e0df8efcccbe7862bf9ec269542aa1c7)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[7289559](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/72895595e0df8efcccbe7862bf9ec269542aa1c7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/concepts/scan-guidance.md](https://github.com/OneDrive/onedrive-api-docs/blob/user/JeremyKelley/ScanGuidance/docs/rest-api/concepts/scan-guidance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online&branch=pr-en-us-961) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C17%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201812172356278109-961%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gYa6MBhGoX9rZs8%2FppTl%2BF8YRUAybj%2BeH81c4Uh9Iuc%3D&st=2018-12-17T23%3A53%3A14Z&se=2019-01-17T23%3A58%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/rest-api/concepts/scan-guidance.md [hunk 1]
```diff
@@ -72,15 +80,26 @@ After your application receives a notification through a webhook, you need to ac
...
+
+### Scanning permissions hierarchies
+
+By default, the delta query response will include sharing information for all items in the query that changed even if they inherit their permissions from their parent and did not have direct sharing changes themselves.  This typically then results in a follow up call to get the permission details for every item rather than just the ones whose sharing information changed.  You can optimize your understanding of how permission changes happen by adding the "Prefer: hierarchicalsharing" header to your delta query request.
+
+When the "Prefer hierarchicalsharing" header is provided, sharing information will be returned for the root of the permissions hierarchy, as well as items that explicitly have sharing changes.  In cases where the sharing change is to remove sharing from an item you will find an empty sharing facet to differentiate between items that inhereit from their parent and those that are unique but have no sharing links.  You will also see this empty sharing facet on the root of a permission hierarchy that is not shared to establish the initial scope.
```
Comment (chackman) [inline docs/rest-api/concepts/scan-guidance.md]: This should be `"Prefer: hierarchicalsharing"` to be consistent with the statement above.

Comment (chackman) [reply docs/rest-api/concepts/scan-guidance.md]: "inherit" rather than "inhereit"

**Comment by daspek** [comment]
OPS Build status updates of commit _[a335b3b](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/a335b3b81a06ae95501f399a45a39d61b580537d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 11 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[a335b3b](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/a335b3b81a06ae95501f399a45a39d61b580537d)_: 

:clock10: **Incremental building**: average incremental building time is _30 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[a335b3b](https://github.com/OneDrive/onedrive-api-docs/pull/961/commits/a335b3b81a06ae95501f399a45a39d61b580537d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/concepts/scan-guidance.md](https://github.com/OneDrive/onedrive-api-docs/blob/user/JeremyKelley/ScanGuidance/docs/rest-api/concepts/scan-guidance.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/scan-guidance?view=odsp-graph-online&branch=pr-en-us-961) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C18%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201812181814236766-961%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Z7GPWHQNDoUrVHTJq0UCYkAYcTVS10zyDRikknLYM%2Fs%3D&st=2018-12-18T18%3A11%3A10Z&se=2019-01-18T18%3A16%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**